### ISSUE TITLE
Placeholder text for not visible rows

### ISSUE THREAD

**Issue by vayurobins** [dillingham/nova-button#47]
Hi.
It would be great if it was possible to have a placeholder text or icon that could sit in those rows where the button->visible = false. Or maybe even show the button as disabled. It looks so empty. :-)
![nova-button](https://user-images.githubusercontent.com/1386354/64620528-52670c00-d3e4-11e9-9d8e-004b449552f5.png)

**Comment by dillingham** [comment]
@vayurobins I agree. Maybe that would be `->disabled(ttrue)` 🤔

**Comment by vayurobins** [comment]
Yes, that could be nice and maybe with som text label for disabled buttons/styles.
Tailwind has a disabled button: https://tailwindcss.com/components/buttons/.
